src/containers/BooksForm.js

```diff
@@ -50,25 +83,34 @@ class BooksForm extends Component {
     ));
 
   render() {
-    const { title, category } = this.state;
+    const { title, category, suggestions } = this.state;
     return (
-      <footer>
+      <div className="form">
         <form onSubmit={this.handleSubmit}>
           <h4>Add new book</h4>
           <div>
             <input
+              list="titles"
               type="text"
-              name="title"
               id="title"
               placeholder="Book title"
               value={title}
-              onChange={this.handleChange}
+              onChange={this.handleOnChange}
+              onKeyUp={this.handleOnKeyUp}
+              aria-label="title"
+              name="title"
             />
+            <datalist id="titles">
+              {suggestions.map(({ id, title }) => (
```

Comment:
Let's use the `id` that we got from Google Books API to get the information from the book when ge save it.